src/components/changeset/user.js

```diff
@@ -117,7 +115,13 @@ export class User extends React.PureComponent {
             </div>
             <div>
               <p className="txt-s color-gray align-center">
-                {`Joined ${registrationDate} | `}
+                {registrationDate != null && (
+                  <React.Fragment>
+                    Joined&nbsp;
+                    <RelativeTime datetime={registrationDate} />
+                    {' | '}
+                  </React.Fragment>
+                )}
```

Comment:
There is a slight behavior change here that I should call out: previously, if for some reason `this.props.userDetails.get('accountCreated')` returned undefined, then the component would display text such as `Joined | 123 edits` (i.e. the string `Joined` would still display, but the date would be omitted). I changed the logic so that instead if the `accountCreated` date can't be retrieved, the user profile just displays the edit count e.g. `123 edits`.